docs/related-work.md

```diff
@@ -18,3 +18,5 @@ Similar tools:
 * [Breezy](https://www.breezy-vcs.org/): Another VCS that's similar in that it
   has multiple storage backends, including its own format as well as .git
   support.
+* [Strudy](https://getsturdy.com/): A Git backed GUI that eliminates local and
```

Comment:
typo in "Strudy" (sorry I didn't notice two days ago)

Reply:
Woops.  Fixed